### DISCUSSION TITLE
区块链使用SM9

### DISCUSSION THREAD

**Discussion by pengpeng1993** [emmansun/gmsm#92] [General]
hello Sun Yimin
我们开源链想使用里面的SM9算法,想知道未来gmsm的规划,以及未来能否让我们加入到这个项目的优化中

**Comment by emmansun** [comment]
基本上，gmsm项目聚焦于国密/商密软件算法的基础实现，具体应用相关的，譬如TLCP等等，我都会建议开发新的项目，这样会更专注一些。对gmsm本身的问题或者优化，欢迎提交Issue/PR。

**Reply by pengpeng1993** [reply]
能和您交换下联系方式吗

**Reply by emmansun** [reply]
> 能和您交换下联系方式吗

邮箱: emman.sun@foxmail.com
